matrix/all-in-one.yaml

```diff
@@ -0,0 +1,534 @@
+    default: ''
+    type: string
+    category: instrumentation
+  - name: OTEL_LOG_LEVEL
+    description: Sets the logging level for instrumentation log messages. Possible values are `none`, `error`, `warn`, `info`, and `debug`. The default value is `info`. Can't be set using the web.config or app.config files.
+    default: info
+    type: enum
+    category: diagnostic logging
+    toDoComment: handle enum values/mark as string
+  - name: OTEL_DOTNET_AUTO_LOG_DIRECTORY
+    description: Directory of the .NET tracer logs. The default value is `/var/log/opentelemetry/dotnet` for Linux, and `%ProgramData%\OpenTelemetry .NET AutoInstrumentation\logs` for Windows. Can't be set using the web.config or app.config files.
+    default: see descruotuin
+    type: string
+    category: diagnostic logging
+    toDoComment: defaultvalue
+  - name: OTEL_DOTNET_AUTO_TRACES_CONSOLE_EXPORTER_ENABLED
+    description: Whether the traces console exporter is activated. The default value is `false`.
+    default: false
+    type: boolean
+    category: diagnostic logging
+  - name: OTEL_DOTNET_AUTO_METRICS_CONSOLE_EXPORTER_ENABLED
+    description: Whether the metrics console exporter is activated. The default value is `false`.
+    default: false
+    type: boolean
+    category: diagnostic logging
+  - name: OTEL_DOTNET_AUTO_LOGS_CONSOLE_EXPORTER_ENABLED
+    description: Whether the logs console exporter is activated. The default value is `false`.The default value is `false`.
+    default: false
+    type: boolean
+    category: diagnostic logging
+  - name: OTEL_DOTNET_AUTO_LOGS_INCLUDE_FORMATTED_MESSAGE
+    description: Whether the log state have to be formatted. The default value is `false`.
+    default: false
+    type: boolean
+    category: diagnostic logging
+  - name: MANUAL SETTINGS
+    description: How to handle https://docs.splunk.com/observability/en/gdi/get-data-in/application/otel-dotnet/configuration/advanced-dotnet-configuration.html#environment-variables-for-manual-installation?
+    default: N/A
+    type: N/A
+    category: manual
+    toDoComment: See description
+instrumentations:
+  - keys:
+      - ASPNET
+    library: ASP.NET Framework (.NET Framework)
+    version: See :ref:`dotnet-otel-versions`
+    comment: MVC / WebApi (Only integrated pipeline mode supported). Metrics requires trace instrumentation.
+    stability: beta
+    supportabilityLevel: community
+    dependencies:
+      - link: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.AspNet
+        version: 1.0.0-rc9.9
+        stability: beta
+      - link: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule
+        version: 1.0.0-rc9.9
+        stability: beta
+    toDoComment: version with ref link
+    signals:
+      - traces: 
+      - metrics: 
+  - keys:
+      - ASPNETCORE
+    library: ASP.NET Core
```

Comment:
I suggest changing to `instrumented-library`

Reply:
4719fc91549726d11c64242ea9eaf57fe1f99b30